pyproject.toml

```diff
@@ -15,6 +15,8 @@ taskipy = "^1.13.0"
 emoji = "^2.12.1"
 yt-dlp = {extras = ["default"], version = "^2024.8.6"}
 telethon = "^1.36.0"
+pytest = "^8.3.2"
+scikit-learn = "^1.5.1"
```

Comment:
Update your branch with the origin main reference, to update this dependencies. And please, add pytest as a dev dependency.